layouts/partials/navbar.html

```diff
@@ -1,6 +1,6 @@
 {{ $cover := .HasShortcode "blocks/cover" }}
 <nav class="js-navbar-scroll navbar navbar-expand navbar-dark {{ if $cover}} td-navbar-cover {{ end }} flex-column flex-md-row td-navbar" data-auto-burger="primary">
-        <a class="navbar-brand img-fluid" href="{{ .Site.Home.RelPermalink }}"></a>
+        <a class="navbar-brand img-fluid" href="{{ .Site.Home.RelPermalink }}" aria-label="Link to home page"></a>
```

Comment:
Can we use a localized string here? See https://github.com/kubernetes/website/pull/48756 for an example of how that might be done.

BTW, I might use `aria-description` rather than `aria-label`. The label text might be Kubernetes. See if you can A/B test this with someone who uses a related accessibility technology and doesn't know our site especially well.